cli/configssh_other.go

```diff
@@ -0,0 +1,5 @@
+//go:build !windows
+
+package cli
+
+var hideForceUnixSlashes = true
```

Comment:
I don't know if it's considered good practice or not, but there's a trick you can use to avoid the build flag:

```go
var hideForceUnixSlashes = func() bool { return runtime.GOOS == "windows" }()
```

I'll leave it up to you whether or not to use it (two less files might be nice). 😄

Reply:
I was following our other pattern. I was unsure if a build tag or that was the move